src/SingleCellAnnotator.imjoy.html

```diff
@@ -1032,8 +1054,9 @@
           <button :disabled="current_image_data.discarded" class="btn transparent-btn" @click="selectLocation(true)" :class="{'selected-location-btn': current_image_data.location[current_image_data.single_location]===true}"><i class="icon icon-check"></i>&nbsp;Yes</button>
           <button :disabled="current_image_data.discarded" class="btn transparent-btn" @click="selectLocation(false)" :class="{'selected-location-btn': current_image_data.location[current_image_data.single_location]===false}"><i class="icon icon-cross"></i>&nbsp;No</button>
           <button :disabled="current_image_index>=cached_image_data.length" class="btn transparent-btn" @click="nextImage()">Next&nbsp;<i class="icon icon-forward"></i></button>
+          <button v-show="current_task_name==='segmentation'" style="background-color:rgb(219, 194, 52)" class="btn btn-sm small-btn channel-controls" @click="deleteAnnotation()">delSingle</button>
         </div>
-        <span v-show="!hide_controls" class="h3 floating-label">{{current_image_data.single_location}}</span>
+        <span v-show="(!hide_controls)&&(current_task_name!=='segmentation')" class="h3 floating-label">{{current_image_data.single_location}}</span>
```

Comment:
Instead of checking the task name here, I think it's better to set a global state `single_cell_mode=true/false`, and use that to change the behaviour.